### PR TITLE
feat: move savekeypair to util

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
Dude just give me write

### CODE REVIEW

File: util/shared.go [hunk 1]
```diff
@@ -170,3 +174,29 @@ func CheckSignature(signature string, pubkey string, message string) (bool, erro
...
+	d, _ = base64.StdEncoding.DecodeString(pubkey)
+	b = pem.EncodeToMemory(&pem.Block{
+		Type:  "DUCKCOIN (ECDSA) PUBLIC KEY",
+		Bytes: d,
+	})
+	if err := ioutil.WriteFile(pubfile, b, 0644); err != nil {
+		return err
+	}
+
+	gchalk.BrightYellow("Your keys have been saved to " + pubfile + "(pubkey) and " + privfile + " (privkey)")
```
Comment (quackduck) [inline util/shared.go]: We don’t want any util functions to print anything, move these prints to below this function’s usage in snodes.

Comment (l3gacyb3ta) [reply util/shared.go]: ok!

File: snode-src/duckcoin.go [hunk 2]
```diff
@@ -286,34 +286,8 @@ func makeKeyPair() (pub string, priv string, err error) {
...
-	}
-
-	gchalk.BrightYellow("Your keys have been saved to " + pubfile + "(pubkey) and " + privfile + " (privkey)")
-	gchalk.BrightRed("Do not tell anyone what's inside " + privfile)
-	return nil
-}
-
 func loadKeyPair(pubfile string, privfile string) (pub string, priv string, err error) {
-	// see comment in util.SaveKeyPair for why the keys are base64 encoded before returning
+	// see comment in util.util.SaveKeyPair for why the keys are base64 encoded before returning
```
Comment (quackduck) [inline snode-src/duckcoin.go]:
```suggestion
	// see comment in util.SaveKeyPair for why the keys are base64 encoded before returning
```

**Comment by quackduck** [comment]
Even if I gave you write, I would still really like you to make PRs which then get reviewed, so it doesn’t really make much of a difference.

**Comment by l3gacyb3ta** [comment]
ofc.

File: util/shared.go [hunk 3]
```diff
@@ -89,6 +93,7 @@ func ToJSON(v interface{}) string {
 	return string(s)
 }
 
+// ArgsHaveOption checks if a given arg sting contains an option
```
Comment (quackduck) [inline util/shared.go]:
```suggestion
// ArgsHaveOption checks command line arguments for an option
```

Comment (l3gacyb3ta) [reply util/shared.go]: Sure!

**Comment by quackduck** [comment]
Looks great!

**Pull request merged**